.github/workflows/nightly-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+name: CI for nightly builds
+
+on:
+  schedule:
+    - cron: "0 8 * * *"
```

Comment:
The `nightly` job should run as part of a larger RAPIDS-wide build pipeline (which will likely live in a separate repository). Therefore we'll want to switch this trigger to use the `workflow_dispatch` event so that it can be triggered programmatically from other pipelines.

- https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch